e2e-tests/playwright/Dockerfile

```diff
@@ -0,0 +1,14 @@
+###############################################################################
+# Dockerfile to create a ready to use Playwright Docker image for end-to-end
+# testing
+###############################################################################
+ARG PLAYWIRGHT_VERSION=1.23.2
```

Comment:
Do not do this. In the future we will have github check for new versions automatically and install them if all tests pass. One reason more why we need the e2e tests

Reply:
This was chosen because the version number for the given image must be fixed (see [here](https://github.com/microsoft/playwright/blob/99d1ad5a88c3e89360829eee92dbaa98d75beaa4/utils/docker/Dockerfile.focal#L5)).
We will solve this with a custom Dockerfile, which also provides the chance to use a slim Debian Image as base instead of a full Ubuntu image.

Reply:
The hardcoded versioning has been fixed in https://github.com/gradido/gradido/pull/2047/commits/196e0e6
Note:
- Since PLaywright developement lacks decent support for Debian or Alpine Linix, it wasnt't worth the effort to create a custom Dockerfile based on either one of them.
- Playwright's required dependencies for the browsers (Chromium, Firefox, and Webkit) are quite a load, including xserver packageges and so forth (as you can see, while building the image from the Dockerfile) - 
> After this operation, 913 MB of additional disk space will be used.
We might take this into consideration for tool choice as well, since we want to get things done.